README.md

```diff
@@ -35,6 +35,25 @@ at the Singapore Haskell meetup: http://typeful.net/talks/hpack
 
 ## Documentation
 
+### Handling of `Paths_` modules
+
+Cabal generates a `Paths_` module fore every package.  By default Hpack adds
+that module to `other-modules` when generating a `.cabal` file.  This is
+sometimes useful and most of the time not harmful.
+
+However, there are situations when this can lead to compilation errors (e.g
+when using a custom `Prelude`).
+
+To prevent Hpack from adding the `Paths_` module to `other-modules` add the
+following to `package.yaml`:
+
+```yaml
+library:
+  when:
+  - condition: false
+    other-modules: Paths_name # substitute name with the package name
```

Comment:
Should we include a note here about how Cabal generates these modules names? I think it replaces hyphens with underscores. I'm not sure if it does anything else.

Reply:
That's the only thing I'm aware of.  Ideally, I would like to link to the Cabal docs on `Path_`, however, I couldn't find anything. 

@phadej do you know if the Cabal docs cover `Paths_` anywhere?